examples/plotting/file/bar_colormapped.py

```diff
@@ -1,3 +1,15 @@
+''' A categorical bar chart illustrating the use of `factor_cmap <factor_cmap>`_ to associate colors from a palette
+with categories.
+
+.. rubric:: Details
+
+:bokeh APIs: :func:`~bokeh.plotting.Figure.vbar`, :func:`~bokeh.transform.factor_cmap`
+:references: :ref:`userguide_categorical_bars`
+:keywords: bar, vbar, legend, factor_cmap, palette
+
+|
```

Comment:
@hyles-lineata, what's this marker for?

Reply:
The pipe acts as a line break and gives a little extra space between the text and the plot.

Reply:
@hyles-lineata I'm not sure why I did't think of this earlier, but we should be able to add space in the `bokeh-plot` directive, so that it won't be necessary to manually add these everywhere. I think remove them for now and make a separate issue fo adding the space in the directive (or I'd be ok making that change in this PR if you want)

Reply:
FYI Eventually I'd like to make a new directive to encapsulate all of this (e.g. including the rubric) 
```
:bokeh-example-metadata:
    :apis: :func:`~bokeh.plotting.Figure.vbar`, :func:`~bokeh.transform.factor_cmap`
    :refs: :ref:`userguide_categorical_bars`
    :words: bar, vbar, legend, factor_cmap, palette